apps/members/migrations/0024_copy_member_email_from_user_model.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.24 on 2021-09-22 18:45
+
+from django.db import migrations
+
+
+def get_email_from_user(apps, schema_editor):
+    Member = apps.get_model('members', 'Member')
+    for member in Member.objects.all():
+        member.email = member.user.email
```

Comment:
La issue original dice que parte del problema es que el email no es obligatorio para user, así que puede haber casos en los que esto sea NULL, pero en el siguiente paso estamos, al menos eso entiendo, poniendo el campo a NOT NULL. Intuyo que esa siguiente migración podría petar en producción si tenemos esos casos.

Reply:
Si, existe ese riesgo pero en la práctica, todos nuestros usuarios tenian email y era unico. Podria darse el problema si desde que lo comprobe alguien se hubiera dado de alta con un email duplicado, pero es poco probable.